1-getting-started.ipynb

```diff
@@ -4,7 +4,7 @@
    "cell_type": "markdown",
```

Comment:
_Commented on notebook `1-getting-started.ipynb` Cell 89 Line 54_
```markdown
* Run a Cell:
        Place your cursor inside the cell and click the Run Cell button that appears next to it.
        Alternatively, use the shortcut `Ctrl + Enter` (Windows/Linux) or `Cmd + Enter` (Mac).

    View Output:
        The output will appear in the Python Interactive window.

2. Option 2: Use `.ipynb` Files
```
The class is gonna by mostly in `.ipynb` files in VSCODE yeah? It might make sense to have that be the first bit of this section. Then list the other options (like py files and jupyter) afterwards.

Or it's probably fine if this is more foundations, then module 2 we can start of inside a notebook!